website/static/js/logTextParser.js

```diff
@@ -142,7 +147,9 @@ var LogText = {
                 })
             ]);
         } else {
-            message = 'There is no text entry in dictionary for the action :' + logObject.attributes.action;
+            message = 'The log viewer has encountered an unexpected log action: ' + logObject.attributes.action +
+                '. Please add a new log entry for this action to logActionsList.js, ' +
```

Comment:
Should be `logActionsList.json`. Not the first time I've seen that cause confusion; it may be best to delete the unused `.js` file.